book/java-rebuilds-service.md

```diff
@@ -0,0 +1,154 @@
+# JVM Build Service
+
+## Overview
+
+The JVM Build Service (JBS) is a controller that will rebuild Java and other JVM Language based dependencies from source.
+
+Java uses a binary distribution model, so the only way to ensure an application is completly built from source is to rebuild all its component libraries from source themselves. Unfortunatly due to the nature of the Java ecosystem this is not an easy process and would normally require a large amount of manual toil.
```

Comment:
Do you want to go into why someone might _want_ to go through the complex/toilsome process of rebuilding everything from scratch?

Reply:
I have added some more context here.